src/services/doppler-legacy-client.ts

```diff
@@ -547,19 +547,22 @@ export class HttpDopplerLegacyClient implements DopplerLegacyClient {
     this.window = window;
     const configRetryCount = 3;
 
-    axiosRetry(this.axios, {
-      retries: configRetryCount,
-      shouldResetTimeout: true,
-      retryCondition: (error) => {
-        return this.isEnabledForRetry(error);
-      },
-      retryDelay: (retryCount: any, error: any) => {
-        if (retryCount === configRetryCount) {
-          logAxiosRetryError(error, addLogEntry);
-        }
-        return retryCount * 700;
-      },
-    });
+    if (!!this.axios) {
+      // TODO: fix this for AppCompositionRootTest in a better way: this allows to continue when axios is undefined
```

Comment:
But, how do you get an undefined axios?

Reply:
In some app tests, it is undefined.

Reply:
Ok, we should fix the tests, but I am fine with the TODO comment.

Reply:
![image](https://user-images.githubusercontent.com/2439363/106294666-3446b980-622e-11eb-8433-bde0fbe2b27d.png)
In AppCompositionRoot tests, because you cannot inject some axios instance here... That's why there's a TODO there.